guides/common/modules/proc_enabling-capsule-in-UI.adoc

```diff
@@ -0,0 +1,33 @@
+[id="assigning-organization-location-capsule-server_{context}"]
+
+= Assigning the Correct Organization and Location to {SmartProxyServer} in the {Project} web UI
+
+After installing the {SmartProxyServer} packages, if there is more than one organization or location, you must assign the organization and location to {SmartProxy} to correctly enable {SmartProxy} in the {Project} web UI.
```

Comment:
we do not enable anything here, we make the Capsule visible. The word enable is missleading.

Reply:
Fixed